mscore/musescore.h

```diff
@@ -22,7 +22,7 @@
 
 #include "config.h"
 #include "globals.h"
-#include "singleapp/src/QtSingleApplication"
+#include "thirdparty/singleapp/src/QtSingleApplication"
```

Comment:
This change looks weird at first, seemingly having nothing to do with this implementation, yet it does. 

From libmscore/cmd.cpp, I utilized mscore()->currentScoreView()->gotoMeasure(); which is what is used in the Find function for finding measures or rehearsal marks from the main menu in MuseScore. I needed to include _mscore.h_ and _scoreview.h_within cmd.cpp to get access to mscore-> and call currentScoreView(), but upon doing this, my compiler wasn't seeing "singleapp/src/QtSingleApplication" anymore, and a quick addition to the actual folder name "thirdparty" before /singleapp solved the issue, and this of course doesn't hurt anything.